### ISSUE TITLE
[3.6.0] Support blocker resize wrong direction

### ISSUE THREAD

**Issue by Festivejelly** [Ultimaker/Cura#5310]
**Application Version**
3.6.0 + (havnt checked older versions)

**Platform**
Windows 10

**Printer**
N/A

**Steps to Reproduce**
Add a support blocker mesh, then go to resize, uncheck uniform resizing and drag the Y hand to scale it on the Y, it instead scales on the Z axis.

**Comment by nallath** [comment]
I've just tried this on master (which is a fair bit ahead of 3.6) and I've not been able to reproduce this. Could you make a video of the bug in action?

**Comment by Ghostkeeper** [comment]
Maybe it happens only after the model itself that the block was linked to was rotated 90 degrees?

**Comment by Ghostkeeper** [comment]
Or it could be another duplicate of this bug report: https://github.com/Ultimaker/Cura/issues/4136

**Comment by Festivejelly** [comment]
Can confirm this is a duplicate of the referenced bug report.
